include/umf/memory_pool_ops.h

```diff
@@ -41,13 +41,72 @@ struct umf_memory_pool_ops_t {
     /// \param pool pool to finalize
     void (*finalize)(void *pool);
 
-    /// Refer to memory_pool.h for description of those functions
+    /// \brief Allocates \p size bytes of uninitialized storage of \p pool
+    /// \param pool memory pool handle
+    /// \param size number of bytes to allocate
+    /// \return Pointer to the allocated memory
+    ///
     void *(*malloc)(void *pool, size_t size);
+
+    /// \brief Allocates memory of \p pool for an array of \p num elements
+    ///        of \p size bytes each and initializes all bytes in the allocated storage
+    ///        to zero
+    /// \param pool memory pool handle
```

Comment:
I think we should drop 'handle' here and perhaps have just "pointer to the memory pool" to avoid confusion. This parameter is NOT the same as pool parameter in the umfPool* functions.